lib/bitex_bot/robot.rb

```diff
@@ -158,12 +157,7 @@ def open_positions?
     end
 
     def sync_closing_flows
-      orders = with_cooldown { Robot.taker.orders }
```

Comment:
aren't needed, it's accessed from taker inside `sync_closed_positions` implementation.